Wikipedia/Categories/UITableViewCell+SelectedBackground.m

```diff
@@ -0,0 +1,14 @@
+//  Created by Monte Hurd on 12/13/14.
+//  Copyright (c) 2014 Wikimedia Foundation. Provided under MIT-style license; please copy and modify!
+
+#import "UITableViewCell+SelectedBackground.h"
+
+@implementation UITableViewCell (SelectedBackground)
+
+- (void)SetSelectedBackground {
```

Comment:
@nirzar naming convention for category methods is: `[wmf_]camelCaseMethodName`.  Categories on classes outside our app have a `wmf_` prefix, and all ObjC methods are camel case.  The prefix prevents potential clashing issues, which can happen as a result of Objective-C's lack of module namespaces in the runtime.  The Swift runtime doesn't have this issue.